src/main/scala/amailp/intellij/robot/psi/reference/PythonKeywordToDefinitionReference.scala

```diff
@@ -24,14 +31,18 @@ class PythonKeywordToDefinitionReference(element: PsiElement, textRange: TextRan
 
   override def multiResolve(incompleteCode: Boolean): Array[ResolveResult] = {
     for {
-      keywordDefinition <- findMatchingInLibraries(element.getText)
+      keywordDefinition <- findMatchingInLibraries(new KeywordReference(element.getText))
     } yield new PsiElementResolveResult(keywordDefinition)
   }.toArray
 
   override def resolve: PsiElement = multiResolve(false).headOption.map(_.getElement).orNull
 
-  def findMatchingInLibraries(reference: String) = {
-    val libraryQNames = currentRobotFile.getImportedLibraries.map(l => QualifiedName.fromDottedString(l.getText))
+  def findMatchingInLibraries(reference: KeywordReference) = {
+
+    var libraries = currentRobotFile.getImportedLibraries
```

Comment:
no `var` please, if not strictly necessary (usually for java compatibility)

Reply:
Now there is a var because it can be redefined later if there is a library name